libs/cardano-cddl/cardano-cddl.cabal

```diff
@@ -0,0 +1,34 @@
+cabal-version:      3.4
+name:               cardano-cddl
```

Comment:
I'd rather avoid adding new packages. Especially tiny packages like this one.

I relive it would be better for `cardano-ledger-conway` to just get an executable section `gen-ccdl` instead, especially since it suppose to generate a file that is part of `cardano-ledger-conway` package itself

Reply:
The idea behind a new package is that there would be a single tool to generate the CDDL for all eras, rather than needing one additional binary per era. It's just that right now, only Conway has been converted. Thoughts?

Reply:
I understand the idea. I am saying that this idea is wrong and doesn't fit with how we structure everything in `cardano-ledger` repo. We do want each era package to be responsible for everything that is going on in that era, especially things like cddl generation. 

So, yes, please create a new executable in `cardano-ledger-conway` instead of a separate package. Later all other eras will get the same kind of executable.